assets/src/block-templates/template-list.js

```diff
@@ -0,0 +1,7 @@
+import * as sideImgTextCta from './side-image-with-text-and-cta';
+
+const templateList = [
+  sideImgTextCta,
+];
+
+export default templateList;
```

Comment:
`export default [ sideImgTextCta ]` it's also fine and you'll avoid to create the [templateList const](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/929/files#diff-fd7451822976d0285f46eb1c7105919fa991e213e25af06a137251a9515a70e2R3).